api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -52,7 +52,7 @@ def update
         authorize! :update, @order, order_token
 
         if @order.update_from_params(params, permitted_checkout_attributes, request.headers.env)
-          if current_api_user.has_spree_role?('admin') && user_id.present?
+          if can?(:admin, @order) && user_id.present?
```

Comment:
from offline conversation it sounds like that is not the case. i have some concern about a user acquiring this role either in base solidus or in a store at some point. is it worth adding an extra warning or spec somewhere around that?